website/docs/language/tests/index.mdx

```diff
@@ -1150,3 +1155,13 @@ testing only uses config from the test file's `backend` blocks and any applicabl
 Finally, [workspaces](https://developer.hashicorp.com/terraform/language/state/workspaces) cannot be used with backends
 used in the context of a test. Any backend used in a test will use the `default` workspace, regardless of
 what workspaces exist and have been selected for use during `plan` and `apply` operations.
+## Skipping Cleanup
+
+You can skip the default [cleanup process](#modules-cleanup) by setting the `skip_cleanup` attribute to `true` in the `test` or `run` block. This is useful for debugging or when you want to inspect the state files after a test run. 
+ When multiple `run` blocks that modify the same state file set this attribute, only the last `run` block with `skip_cleanup` set to `true` will take effect. This means the state file will reflect the state as of that last `run` block.
+ Any resources retained in the state due to this attribute will also be saved in the state file artifacts. Refer to the [Artifacts](#handling-leftover-state-artifacts) section for more information. This means that you have to cleanup the state before running any other tests.
+
+## Handling Leftover State Artifacts
+
+During the execution of a test file, Terraform automatically cleans up any in-memory state it generates. This cleanup involves destroying the created remote resources. However, if the cleanup process fails, Terraform will persist a manifest file that references state files containing the remaining resources. The generated files are located within the test directory of the terraform configuration (`.terraform/test`).
+These lingering resources will block future Terraform test operations and must be manually removed to proceed.
```

Comment:
@SarahFrench I updated the section and made mention only to the test directory as a whole, not the structure.

Reply:
Perfect, then disregard my last comment on the other thread